python/lsst/sdm/schemas/apdb.yaml

```diff
@@ -1017,11 +560,25 @@ tables:
     datatype: float
     description: Mean of the y band flux errors.
     fits:tunit: nJy
+<<<<<<< HEAD
```

Comment:
These merge conflict markers are removed in your next commit, but it would be good to have each commit be clean.